src/test/java/divaeva/testNG/hw/RegistrationNewUserTest.java

```diff
@@ -0,0 +1,98 @@
+package divaeva.testNG.hw;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class RegistrationNewUserTest {
+    private final By REGISTRATION_BUTTON_ON_HOME_PAGE = By.xpath("//rz-user[@class='header-actions__component']" +
+            "//button[@class='header__button ng-star-inserted']");
+    private final By REGISTER_BUTTON = By.xpath("//button[text()=' Зареєструватися ']");
+    private final By EMPTY_REGISTRATION_FORM_FIELDS = By.xpath("//div[contains(@class,'type_error')]");
+    private final By VALIDATION_ERROR_MESSAGES = By.xpath("//form-error[@class='validation-message']");
```

Comment:
xpath should be changed. It finds 4 elements, even those that aren't visible. In fact it should find 2 elements
![Screenshot_3](https://github.com/AnastasiaDivaeva/TestNGHomework/assets/66726924/d692ae46-d857-4353-8c99-a10daea7659f)